### DISCUSSION TITLE
In live web app, I am getting e.removeHeader is not a function at SendStream.removeContentHeaderFields

### DISCUSSION THREAD

**Discussion by Sawan1994** [vercel/next.js#25963] [Help]
Getting below error in nextjs v10 application on live website frequently : 

TypeError: e.removeHeader is not a function at SendStream.removeContentHeaderFields (/usr/src/app/node_modules/next/dist/compiled/send/index.js:1:52146) at SendStream.notModified (/usr/src/app/node_modules/next/dist/compiled/send/index.js:1:52259) at SendStream.send (/usr/src/app/node_modules/next/dist/compiled/send/index.js:1:54962) at onstat (/usr/src/app/node_modules/next/dist/compiled/send/index.js:1:55900) at Shim.applySegment (/usr/src/app/node_modules/newrelic/lib/shim/shim.js:1430:20) at wrappedCallback (/usr/src/app/node_modules/newrelic/lib/shim/shim.js:1287:21) at FSReqCallback.oncomplete (fs.js:193:5) at FSReqCallback.callbackTrampoline (internal/async_hooks.js:134:14)

**Comment by devNimi** [comment]
it's hard to reproduce
happening for me also in a production app
@timneutkens

**Reply by olivertilk** [reply]
I started seeing this error too in dev and start mode, but only when using the 'basepath' property in my next.config.js:

`module.exports = {
  basePath: '/dashboard',
}`

**Reply by CanKattwinkel** [reply]
Seeing the same issue, hit us out of the blue on a live application. We noticed that this keeps happening to 3 containers of the same application **at the same time**.  This kinda supports the related [claim here](https://github.com/vercel/next.js/issues/19309#issuecomment-874402983). I will update if we find any new information. To us, this is very bad as it happens in production and not in test environments.

**Reply by CanKattwinkel** [reply]
Applied temporary patch on my end as described here https://github.com/vercel/next.js/issues/19309#issuecomment-891087160 @MrNorm FYI

**Reply by MrNorm** [reply]
Appreciate the mention @CanKattwinkel - Taking a look now

**Reply by jedrker** [reply]
The same here. Thanks, @CanKattwinkel for the patch!
